pom.xml

```diff
@@ -213,6 +213,7 @@
     <javax.servlet-api.version>4.0.1</javax.servlet-api.version>
     <jakarta.annotation-api.version>1.3.5</jakarta.annotation-api.version>
     <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
+    <jspecify.version>1.0.0</jspecify.version>
```

Comment:
Is this a Java EE libarary? If not, let's keep it separately for easier management